### PR TITLE
Updated NewRelic instructions.

### PR DESCRIPTION
**Pull request by glogiotatidis**
This unit file mounts /proc /dev and /sys inside the NewRelic container
which allows disk, network and process reporting. The Dockerfile used is
the same as the one we replace.

### CODE REVIEW

**Comment by glogiotatidis** [comment]
@jgmize r?

File: README.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 ### Add NewRelic Server Monitoring
 
 1. SSH to one of the hosts
-2. git clone https://github.com/johanneswuerbach/newrelic-sysmond-service
-3. Edit `newrelic-sysmond.service` and replace `YOUR_NEW_RELIC_KEY` with our NewRelic Key and `CUSTOM_HOSTNAME=%H` with `CUSTOM_HOSTNAME=webprod-deis-%H`
+2. git clone https://github.com/lorieri/coreos-newrelic
+3. Edit `newrelic-sysmond.service` and replace `YOUR_NEW_RELIC_KEY` with our NewRelic Key and `-n %H` with `-n webprod-deis-%H`
```
Comment (jgmize) [inline README.md]: Why not copy the service file into this repo with a comment about the original source? Then we can apply tweaks like custom hostname prefixes directly.

**Comment by glogiotatidis** [comment]
good point @jgmize. updated

**Comment by jgmize** [comment]
Thanks @glogiotatidis!

**Pull request merged**